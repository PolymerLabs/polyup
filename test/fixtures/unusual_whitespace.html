<polymer-element name="foo">
  <template><style>
    </style>
  </template>
</polymer-element>
