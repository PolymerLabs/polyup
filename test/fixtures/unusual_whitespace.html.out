<dom-module id="foo">
  <style>
    </style>
  <template>
  </template>
</dom-module>
